frontend/src/scenes/web-analytics/WebAnalyticsHealthCheck.tsx

```diff
@@ -3,6 +3,8 @@ import { LemonBanner } from 'lib/lemon-ui/LemonBanner'
 import { Link } from 'lib/lemon-ui/Link'
 import { ConversionGoalWarning, webAnalyticsLogic } from 'scenes/web-analytics/webAnalyticsLogic'
 
+const KNOWN_SERVER_SIDE_SDKS = ['segment', 'posthog-node', 'posthog-ruby', 'posthog-go', 'posthog-php', 'posthog-java']
```

Comment:
I didn't include rust and elixir as they don't send the $lib property

Reply:
Should we update them to include it?

Reply:
Yeah, though in practice, the ones in this list will cover 99% of users.